### ISSUE TITLE
Bug Report - when a new resource is created this policy (definition es deploy asc security contacts) doesn't trigger a remediation as expected

### ISSUE THREAD

**Issue by bahramsabri** [Azure/Enterprise-Scale#997]

**Comment by bahramsabri** [comment]
@krowlandson By default the remediation kicks in 10 minutes after resource deployment. Do you think this on demand evaluation scan will help us to fix the issue we have with this policy?

**Comment by bahramsabri** [comment]
@krowlandson Yes, I have tried waiting 30 minutes and even more and resource type is microsoft.resources/subscriptions. The policy works on existing and it does reports 1 non-compliant resource and 28 compliant resources and if I create a new remediation task using Terraform, the policy will fix the non-compliant resource. The problem is when a new resource is created the policy doesn't trigger a remediation as expected.

**Comment by jtracey93** [comment]
Trigger ADO Sync 1

**Comment by jtracey93** [comment]
Trigger ADO Sync 2
